README.md

```diff
@@ -47,81 +28,62 @@ Use the following dependency in your project to grab via Maven:
-If you want to build your own .jar, execute the following from within the cloned directory:
+If you want to build your own `.jar`, execute the following from within the cloned directory:
+
 ```shell
   mvn package
 ```
 
-### Environment Variables
+## Supported Java versions 
 
-You can use Environment Variables for storing configuration like application token or base URL rather than hardcoding them into your application.
-
-| Name                              | Description                                                                                                      |
-|-----------------------------------|------------------------------------------------------------------------------------------------------------------|
-| `EMNIFY_BASE_PATH`                  | the base URL to form a request. Default value:  `https://cdn.emnify.net`                                         |
-| `EMNIFY_APPLICATION_TOKEN`          | variable for authentication via application token. [more](https://cdn.emnify.net/api/doc/application-token.html) |
-| `EMNIFY_USERNAME` and `EMNIFY_PASSWORD` | variable for authentication via username and password. [more](https://cdn.emnify.net/api/doc/basic-auth.html)      |
-
-### Examples
+This library supports the following Java implementations:
 
-#### Initialize the Client using environment variables
+- [OpenJDK 8](https://openjdk.org/projects/jdk8/)
+- [OpenJDK 11](https://openjdk.org/projects/jdk/11/)
+- [OracleJDK 8](https://www.oracle.com/java/technologies/downloads/#java8)
+- [OracleJDK 11](https://www.oracle.com/java/technologies/downloads/#java11)
 
-```java
-    EMnify.authenticate();
-```
+## TLS requirements
 
-#### Initialize the Client via passing authorization parameter
+Refer to the [SSL Report table](https://www.ssllabs.com/ssltest/analyze.html?d=cdn.emnify.net&latest) for the supported Transport Layer Security (TLS) versions.
 
-```java
-    String username = "user@domain.com";
-    String password = "2fd4e1c67a2d28...";
-    
-    EMnify.authenticate(username, password)
-```
+## Environment variables
 
-#### Initialize the Client via application token
+You can use environment variables for storing configuration like application token or base URL rather than hardcoding them into your application.
 
-```java
-    String applicationToken = "KAOp24TuMgjO2FpZmZ3ZFjSqpk7ea_mY8...";
-    
-    EMnify.authenticate(applicationToken)
-```
+| Name               | Description                                                              |
+|--------------------|--------------------------------------------------------------------------|
+| `EMNIFY_BASE_PATH` | Base URL to form a request. Default value:  `https://cdn.emnify.net` |
+| `EMNIFY_APPLICATION_TOKEN`  | Variable for authentication via [application token](https://cdn.emnify.net/api/doc/application-token.html). |
+| `EMNIFY_USERNAME` and `EMNIFY_PASSWORD` | Variable for authentication via [username and password](https://cdn.emnify.net/api/doc/basic-auth.html). |
```

Comment:
See lines 75 and 76 in the [product docs](https://github.com/EMnify/product-docs/blob/69d32508a0dd23cf7c2a0cd14cdf4d2fbaf26f41/docs/sdks/java/getting-started.md?plain=1#L76)

```suggestion
| `EMNIFY_APPLICATION_TOKEN`  | Variable for authenticating via an [application token](https://cdn.emnify.net/api/doc/application-token.html). |
| `EMNIFY_USERNAME` and `EMNIFY_PASSWORD` | Variables for authenticating via [username and password](https://cdn.emnify.net/api/doc/basic-auth.html). |
```